mitosheet/mitosheet/tests/updates/test_update_existing_import.py

```diff
@@ -559,3 +559,50 @@ def test_update_imports_replays_unchanged_files_correctly_from_analysis_name():
     # Remove the test files
     os.remove(TEST_CSV_FILE)
     os.remove(TEST_EXCEL_FILE)
+
+
+@pandas_post_1_4_only
+@python_post_3_6_only
+def test_update_user_defined_imports():
```

Comment:
Let's add a test for:
1. Updating non custom import to custom import
2. Updating custom import to non-custom import